functions/.eslintrc.js

```diff
@@ -36,7 +36,9 @@ module.exports = {
     '@typescript-eslint/explicit-module-boundary-types': 'off',
     '@typescript-eslint/no-unsafe-return': 'off',
     '@typescript-eslint/no-var-requires': 'off',
+    '@typescript-eslint/restrict-template-expressions': 'off',
```

Comment:
Which part was affected by this rule?

Reply:
these [line](https://github.com/meilisearch/firestore-meilisearch/blob/logger/functions/src/logs.ts#L55), [here](https://github.com/meilisearch/firestore-meilisearch/blob/logger/functions/src/logs.ts#L67), [here](https://github.com/meilisearch/firestore-meilisearch/blob/logger/functions/src/logs.ts#L78):
```
    `Creating new document ${id} in MeiliSearch index ${process.env.MEILISEARCH_INDEX_NAME}`,
```
With this eslint error:
```
Invalid type "string | undefined" of template literal expression.
```

Reply:
```
export function addDocument(id: string, data: object) {
  logger.info(
    `Creating new document ${id} in MeiliSearch index ${process.env.MEILISEARCH_INDEX_NAME}`,
    data
  )
}
```

What type is `string | undefined` ?

Reply:
This variable: ` ${process.env.MEILISEARCH_INDEX_NAME}`